frame/support/procedural/src/lib.rs

```diff
@@ -666,18 +667,13 @@ pub fn derive_runtime_debug_no_bound(input: TokenStream) -> TokenStream {
 			const _: () = {
 				impl #impl_generics core::fmt::Debug for #name #ty_generics #where_clause {
 					fn fmt(&self, fmt: &mut core::fmt::Formatter) -> core::fmt::Result {
-						fmt.write_str("<stripped>")
+						fmt.write_str("<wasm:stripped>")
```

Comment:
the docs for `derive_runtime_debug_no_bound` needs to be updates, it refers to `<stripped>` string.